final_prep/mod_zero_hero.rb

```diff
@@ -47,38 +62,79 @@
 # assess_situation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
 #assess_situation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
 #assess_situation(3, announcement, excuse) > should print - "Meh. Hard pass."
-
+assess_situation(99, announcement, excuse)
+assess_situation(21, announcement, excuse)
+assess_situation(3, announcement, excuse)
 # Declare a new variable - scary_monster - assigned to an hash with the following key/values
 #   - name (string)
 #   - smell (string)
 #   - weight (integer)
 #   - citiesDestroyed (array)
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
-
+scary_monster = {
+  'name' => 'Alex the OK',
+  'smell' => 'stanky',
+  'weight' => 180,
+  'citiesDestroyed' => ['Austin', 'Houston', 'Dallas'],
+  'luckyNumbers' => [5, 6, 7],
+  'address' => {'number' => 5200,
+    'street' => 'East Jefferson Avenue',
+    'state' => 'TX',
+    'zip' => 77963}
+}
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+class SuperHero
+  def initialize(name, super_power, age)
+    @name = name
+    @super_power = super_power
+    @age = age
+  end
+  def arch_nemesis
+    p "The Syntax Error"
+  end
+  def power_level(number)
```

Comment:
This method takes an argument, but then does not use that. You may not need anything in the parens here.

Reply:
I've gotten rid of the parenthesis (number).